tests/integration/article.test.js

```diff
@@ -0,0 +1,73 @@
+import chaiHttp from "chai-http";
```

Comment:
@Musigwa, bro!
It looks like this feature is not heavily tested.
Would you consider adding more test cases?
Eg : 
- what happens when you are creating an article and title is not provided?
- what happens when you are creating a post and body is not provided?
- what happens when you are creating a post and description is not provided?
- what result are you expecting after creating a post?